apps/api/src/main/java/dk/treecreate/api/designs/dto/CreateDesignRequest.java

```diff
@@ -0,0 +1,44 @@
+package dk.treecreate.api.designs.dto;
+
+import dk.treecreate.api.designs.DesignType;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+public class CreateDesignRequest
+{
+    @NotNull
+    @ApiModelProperty(example = "{ title: Example design," +
+        "  front: 'Roboto'," +
+        "  design: 'first'," +
```

Comment:
is title: the title of the design? If so Example design should be in quotes. font, not front. What is design: 'first' if the title is the title that the user writes

Reply:
Resolved in e65dbb6

The exact format the data will be saved will most likely change once I fully implement saving